MeshGeoToolsLib/GeoMapper.cpp

```diff
@@ -433,8 +433,13 @@ getCandidateElementsForLineSegmentIntersection(
 }
 
 static bool snapPointToElementNode(MathLib::Point3d& p,
-                                   MeshLib::Element const& elem, double sqr_eps)
+                                   MeshLib::Element const& elem, double rel_eps)
 {
+    // values will be initialized within computeSqrNodeDistanceRange
+    double sqr_min, sqr_max;
+    elem.computeSqrNodeDistanceRange(sqr_min, sqr_max);
+
+    double const sqr_eps(rel_eps*rel_eps * sqr_min);
```

Comment:
So now if e.g. `rel_eps = 1e-3`, then the tolerance in distances is 1e-3 times the minimum edge length of the element? Sounds good.
Note: Now rel_eps is not a tolerance of "distance squared" anymore, but of "distance".